lib/webrtc/util/eventtarget.js

```diff
@@ -1,41 +1,41 @@
-'use strict';
+// 'use strict';
 
-const EventEmitter = require('events').EventEmitter;
+// const EventEmitter = require('events').EventEmitter;
 
-/**
- * Event target.
- * @class
- */
-class EventTarget {
-  constructor() {
-    Object.defineProperties(this, {
-      _eventEmitter: {
-        value: new EventEmitter()
-      }
-    });
-  }
+// /**
+//  * Event target.
+//  * @class
+//  */
+// class EventTarget {
+//   constructor() {
+//     Object.defineProperties(this, {
+//       _eventEmitter: {
+//         value: new EventEmitter()
+//       }
+//     });
+//   }
```

Comment:
If we no longer need this copy of EventTarget, please remove this as well. Since I believe we are now using the EventTarget class that lives [here](https://github.com/twilio/twilio-video.js/blob/00ec520fc68f3065f94e541d61e7736a0bba06b5/lib/eventtarget.js#L5)?